packages/cli/src/utils/exchange.ts

```diff
@@ -37,11 +37,11 @@ export async function checkNotDangerousExchange(
     quotedAmountToReceiveWithBuffer,
     oracleMedianRate
   )
-  if (expectedSlippage > depeggedPricePercentage) {
+  if (Math.abs(expectedSlippage) > depeggedPricePercentage) {
```

Comment:
Do we have any guarantees that `depeggedPricePercentage` is positive?
eg `Math.abs(expectedSlippage) > Math.abs(epeggedPricePercentage)` may be safer?

Reply:
its actually defined as a const here https://github.com/celo-org/developer-tooling/blob/master/packages/cli/src/exchange-stable-base.ts#L11 and then passed into the function as an arg. 

so its safe in that since

Reply:
Nitpick then: why are we using it as an arg if it's a constant, not relevant for merging this PR though

Reply:
gonna rename the const